docs/products/postgresql/howto/ai-insights.md

```diff
@@ -0,0 +1,15 @@
+---
+title: AI database optimizer for Aiven for PostgreSQL®
+sidebar_label: AI database optimizer
+keywords: ["AI", "Artificial intelligence", "Aiven AI Database Optimizer"]
+description: "Use Aiven's artificial intelligence capabilities to identify slow queries and get optimization suggestions."
+---
+
+import AIInsights from "@site/static/includes/ai-insights.md"
+
+<AIInsights service="Aiven for PostgreSQL®"/>
+
+## Related pages
```

Comment:
ToC for this page shows one section only: Related pages. I guess we better hide it from the ToC unless we have some some sections to include.